packages/TelegramClient-UI.package/TCUChatListItem.class/instance/addTitle.st

```diff
@@ -1,8 +1,13 @@
 drawing
 addTitle
 
+	| prefix |
+	prefix := ''.
+	self chat isPinned ifTrue: [
+		prefix := '[pinned] '
+	].
 	self addMorph: (TextMorph new
 		position: self topLeft + self class defaultTitleOffset;
-		contents: ((self chatName condensedIntoOneLine truncateWithElipsisTo: self class defaultTitleLength) asText addAttribute: TextEmphasis bold);
+		contents: ((prefix, self chatName condensedIntoOneLine truncateWithElipsisTo: self class defaultTitleLength) asText addAttribute: TextEmphasis bold);
```

Comment:
`truncateWithElipsisTo:` doesn't work because defaultTitleLength should've gotten shorter with the addition of profile pictures and pinned messages. However, it didn't work before either, so it shouldn't be a problem now.